arm/cores/HardwareSerial.cpp

```diff
@@ -211,9 +211,13 @@ while( XMC_USIC_CH_GetTransmitBufferStatus( _XMC_UART_config->channel) == XMC_US
 size_t HardwareSerial::write( const uint8_t uc_data )
 {
 // Is the hardware currently busy?
+#if defined (XMC4700_Radar_Baseboard)
```

Comment:
Conditional compiling for each of the board variants at this level should be avoided.

Reply:
Wrong way rouns as that is how Arduino Refernces are done for all of their boards with minimal in core level